src/quo2/components/settings/accounts/view.cljs

```diff
@@ -0,0 +1,46 @@
+(ns quo2.components.settings.accounts.view
+  (:require [quo.react-native :as rn]
```

Comment:
react-native.core :as rn

Reply:
Solved in 3d9a6bf0afe3430696defb1e18fa8de2d9a94eaa